Src/LiquidProjections/MapBuilding/IAction.cs

```diff
@@ -88,6 +123,18 @@ public interface ICrudAction<TEvent, TProjection, TKey, TContext> : IAction<TEve
         [Obsolete("Use AsUpdateOf().IgnoringMissing() instead")]
         IUpdateAction<TEvent, TKey, TProjection, TContext> AsUpdateIfExistsOf(Func<TEvent, TKey> getKey);
 
+        /// <summary>
+        /// Continues configuring a handler for events of type <typeparamref name="TEvent"/>.
+        /// Specifies that the projection with the specified key will be updated by the event handler.
+        /// The event will not be handled by the handler if the projection with such key does not exist.
+        /// </summary>
+        /// <param name="getKey">The delegate that determines the projection key for the event.</param>
+        /// <returns>
+        /// <see cref="IUpdateAction{TEvent,TKey, TProjection,TContext}"/> that allows to continue configuring the handler.
+        /// </returns>
+        [Obsolete("Use AsUpdateOf().IgnoringMissing() instead")]

```

Comment:
Let's not add additional overloads that are obsolete.

Reply:
I agree, will remove 👍